### DISCUSSION TITLE
Encontro FrontendBR - BrazilJS FORTAL

### DISCUSSION THREAD

**Discussion by cjpatoilo** [frontendbr/forum#687] [Discussão]
**Olá pessoal!**

Como várias pessoas daqui já participaram do encontro da BrazilJS em Porto Alegre e sabem que há encontro do grupo @frontendbr nas edições, decidir organizar e manter a tradição do nosso encontro na próxima edição da BrazilJS em Fortaleza. 

Será na quinta-feira (pré conferência) às 19:00h no [Boozer's Pub](http://www.boozers.com.br/) [[map](https://goo.gl/maps/pVdRZH2tBQ62)]. 

Eu já estou em Fortaleza e espero conhecer vocês
Abraços

_Nota: Se houver a necessidade de um lugar maior esse post será atualizado informando o nome e as coordenadas do lugar._

**Comment by thompsonemerson** [comment]
Só bora! 🚀  🍺

**Comment by IanRamosC** [comment]
Topo hein. :stuck_out_tongue_winking_eye:

**Comment by eduardojmatos** [comment]
Boa! Pena eu não ir, mas boa pré-BrazilJS! =D

**Comment by iamtchelo** [comment]
#partiu

**Comment by munizart** [comment]
@diogomoretti @rricardi @viniciuskneves @owliveira

**Comment by joeljuca** [comment]
Bora!

**Comment by felipefialho** [comment]
Fiquei afim de ir em Fortaleza, quem sabe na próxima :)

**Comment by jdrgoms** [comment]
Vou ficar devendo dessa vez @thompsonemerson @cjpatoilo  :cry:

**Comment by renanrboliveira** [comment]
Já é!

**Comment by jamile-dev** [comment]
Vumbora! Tô chegando amanhã :)

**Comment by IanRamosC** [comment]
Que legal, galera. Espero que aqueles que não podem ir amanhã possam vir na próxima vez.

**Comment by yannbf** [comment]
Partiu :tada:

**Comment by jeancsanchez** [comment]
Cuida negrada!

**Comment by FernandaBernardo** [comment]
Bora!

**Comment by IanRamosC** [comment]
Já tou aqui e só tem eu. Hahaha

**Comment by jeancsanchez** [comment]
To chegando!

**Comment by yannbf** [comment]
À caminho!

Em 31 de ago de 2017 7:23 PM, "Ian Ramos" <notifications@github.com>
escreveu:

**Comment by cjpatoilo** [comment]
![21284366_1403695043011048_829503343_o](https://user-images.githubusercontent.com/1542831/29955934-d89d3032-8eb9-11e7-9799-17a09ac4171b.jpg)

Pessoal, prazer imenso conhecer todos vocês. 
Bom evento para todos! 😊

**Comment by cassiodourado1** [comment]
fui embora antes da foto... rs
